services/blockchain-connector/shared/sdk/index.js

```diff
@@ -125,6 +125,8 @@ const init = async () => {
 	await cacheFeeConstants();
 	await updateTokenInfo();
 
+	await verifyClientConnection();
```

Comment:
This may not be necessary. The verification logic can be an IIFE within `client.js`.